### PR TITLE
Ft busstop crud

### PR DESCRIPTION
**Pull request by Imaraika**
#### What does this PR do?
Creating a CRUD operation for buseStop

#### Description of Task to be completed?
- Creating a database
- Using sequelize to generate models, migrations, and seeders
- Creating all bus Stops
- Testing code using Jest
#### How should this be manually tested?
Use postman to test bus endpoints
#### Any background context you want to provide?
User should be able to
 - CREATE BusStops
 - GET BusStops
 - GET BusStops
 - UPDATE BusStops
 - DELETE BusStops

#### What are the relevant trello boards stories?
 - https://trello.com/c/ECnui8v9/110-crud-operations-for-busstop
#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A

### CODE REVIEW

**Comment by Imaraika** [comment]
ft(ft-busstop-crud) the CRUD operation for BusStop have been implemented

File: src/database/migrations/20201001230119-create-bus-stop.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  // eslint-disable-next-line no-unused-vars
```
Comment (malaba6) [inline src/database/migrations/20201001230119-create-bus-stop.js]: stop disabling eslint rules

File: src/database/models/index.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ fs
   .filter((file) => (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js'))
   .forEach((file) => {
     // eslint-disable-next-line global-require
+    // eslint-disable-next-line import/no-dynamic-require
```
Comment (malaba6) [inline src/database/models/index.js]: same here

**Review by malaba6 — CHANGES_REQUESTED** [review]

File: src/database/models/index.js (hunk 2 above)
Comment (malaba6) [reply src/database/models/index.js]: You must write unit tests for your functions

**Comment by Rwandarushya** [comment]
As you validated the input for creating bus Stop you should also validate your inputs when you are updating the bus stop 
Screen Shot 2020-10-06 at 19 52 55

**Review by Rwandarushya — COMMENTED** [review]
add the validation on the update bus stop route
Screen Shot 2020-10-06 at 19 57 18

File: src/database/migrations/20201001230119-create-bus-stop.js (hunk 1 above)
Comment (Imaraika) [reply src/database/migrations/20201001230119-create-bus-stop.js]: yes yes! Thank you for thee feedback I have worked on this!

File: src/database/models/index.js (hunk 2 above)
Comment (Imaraika) [reply src/database/models/index.js]: the unit tests have been added

**Comment by Imaraika** [comment]
[CRUD operations for BusStop](https://trello.com/c/ECnui8v9/110-crud-operations-for-busstop)

**Review by Monehin — APPROVED** [review]

**Pull request merged**